lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/audio_output.rb

```diff
@@ -0,0 +1,70 @@
+class Mic
+end# -*- coding: binary -*-
+require 'rex/post/meterpreter'
+require 'bindata'
+
+module Rex
+  module Post
+    module Meterpreter
+      module Ui
+
+        ###
+        #
+        # Play audio on remote system
+        #
+        ###
+        class Console::CommandDispatcher::Stdapi::AudioOutput
+          Klass = Console::CommandDispatcher::Stdapi::AudioOutput
+
+          include Console::CommandDispatcher
+
+          #
+          # List of supported commands.
+          #
+          def commands
+            all = {
+              'play' => 'play an audio file on target\'s system, nothing written on disk'
```

Comment:
I'd suggest using double quotes for the description portion so you could drop the escape. 
Per [The Ruby Style Guide - Strings](https://github.com/bbatsov/ruby-style-guide#strings):
>Prefer double-quotes unless your string literal contains " or escape characters you want to suppress.

Reply:
`target system` also works

Reply:
Rewording is fine as well. I figured I'd use this as an opportunity to introduce the style guide. ;)

Reply:
Indeed! I have taken the rewording way, will keep in mind the style guide for the upcoming coding sessions.